public/app/features/dashboard/state/PanelModel.ts

```diff
@@ -324,8 +324,8 @@ export class PanelModel {
     return this.queryRunner;
   }
 
-  hasTitle(): boolean {
-    return this.title && this.title.length > 0;
+  hasTitle() {
+    return !!this.title.trim();
```

Comment:
wont this treat '0' as false?

Reply:
nope since it's a string. It would if title was a number